recipes/jupyter-syncthing-proxy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jupyter-syncthing-proxy" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter-syncthing-proxy-{{ version }}.tar.gz
+  sha256: df3d74d140e97ed73a5e4c3862564799dd92c53e8cfd83c8ef29a5d95c82dc27
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - jupyter-server-proxy
+    - python >=3.8
```

Comment:
Should https://anaconda.org/conda-forge/syncthing also be added as a dependency?

Reply:
Oo, that would be good. To be honest, I have no idea what this package does :laughing:

Reply:
Done at bb95e5ff2dbcf3dfb50a0ecc8ebbab8f13271fe8!